dist/routable-component.js

```diff
@@ -455,17 +455,17 @@ var routableComponent =
 	  }, {
 	    key: 'event',
 	    value: function event() {
-	      if (this._current_from && this._current_controller) {
+	      if (this._currentFrom && this._currentController) {
 	        if (arguments.length > 2) throw new Error('arguments too long');
-	        this.event_on_from_controller.apply(this, arguments);
-	      } else if (this._current_from) {
+	        this.eventOnFromController.apply(this, arguments);
+	      } else if (this._currentFrom) {
 	        if (arguments.length > 3) throw new Error('arguments too long');
-	        this.event_on_from.apply(this, arguments);
-	      } else if (this._current_controller) {
+	        this.eventOnFrom.apply(this, arguments);
+	      } else if (this._currentController) {
```

Comment:
__[eslint]__
* Error - Mixed spaces and tabs. (no-mixed-spaces-and-tabs)

<!-- Comment by [SideCI](https://sideci.com) -->